### DISCUSSION TITLE
TractSeg

### DISCUSSION THREAD

**Discussion by mgreen-neura** [neurodesk/neurodesk.github.io#97] [General]
Hi team, just wondering if there are any plans to include TractSeg into NeuroDesk?
https://github.com/MIC-DKFZ/TractSeg

**Comment by stebo85** [comment]
Yea, that’s on the list of planned tools to be added. Would you like to add it and propose a pull request to help?

**Comment by stebo85** [comment]
here is the open issue: https://github.com/NeuroDesk/neurocontainers/issues/37

**Comment by stebo85** [comment]
and the successor to it: https://github.com/NeuroDesk/neurocontainers/issues/767
